package.json

```diff
@@ -7,10 +7,11 @@
   "scripts": {
     "setup": "lerna bootstrap",
     "build": "yarn util:build",
+    "build:core": "lerna exec --scope @hoprnet/hopr-core yarn build",
     "lint": "npx prettier --check .",
     "lint:fix": "npx prettier --write .",
     "test": "lerna run test --stream",
-    "test:core": "lerna exec --scope @hoprnet/hopr-core yarn build && lerna exec --scope @hoprnet/hopr-core yarn test",
+    "test:core": "yarn build:core && lerna exec --scope @hoprnet/hopr-core yarn test",
```

Comment:
Why is this in the logging PR?

Reply:
This makes development easier. Having `build:core` saves on my machine around 16 seconds per recompile.

Reply:
that's fine, it should be a separate PR.